web/spacewalk-web.spec

```diff
@@ -46,6 +46,7 @@ Source1:        node-modules.tar.gz
 BuildRoot:      %{_tmppath}/%{name}-%{version}-build
 BuildArch:      noarch
 Requires(pre):  uyuni-base-common
+BuildRequires:  gettext
```

Comment:
Did you verify that this works with openSUSE?

Reply:
I did via copr: https://copr.fedorainfracloud.org/coprs/sbluhm/uyuni-server/build/4351686/
`Gettext` should be required by both as the binary gets called.. It seems that OBS is installing more additional packages than COPR.